.github/workflows/on-push-release.yml

```diff
@@ -22,3 +22,26 @@ jobs:
       server_version: main
       ubuntu_version: ${{ matrix.ubuntu_version }}
       compiler: ${{ matrix.compiler }}
+
+  dev-mode:
+    name: "DEV_MODE"
+    uses: ./.github/workflows/build-test.yml
+    with:
+      dev_mode: ON
+      server_version: main
+      verbose_test_output: ON
+      verbose_make_output: ON
+
+  sanitize-addr:
+    name: "Sanitize address"
+    uses: ./.github/workflows/build-test.yml
+    with:
+      sanitize: address
+      server_version: main
+
+  san-thread:
```

Comment:
nit: Not sure why you would shorten to `san-` here while you used `santize-` for the address matrix run above :-)

Reply:
shorter is better? :) (will change for consistency)